apps/src/templates/sectionAssessments/assessmentsTestHelpers.js

```diff
@@ -1,5 +1,6 @@
 import i18n from '@cdo/locale';
 
+// Data for students assessments multiple choice table
```

Comment:
is this data for multiple students? if it is, this should say "Data for _students'_ assessments multiple choice table"

Reply:
Fixed.  Thanks